index.html

```diff
@@ -174,8 +174,40 @@ <h3>LET'S  CONNECT</h3>
                 <li><img src="./image/imgg1.png" alt="line2"></li>
             </ul>
             </div>
+            <div class="formy">
+                 <center> <div class="info-form">
+                    <h2 class="h002">Contact me</h2>
+                    <h3 class="h003">If you have an application you
+                         are interested in developing,
+                          a feature that you need
+                         built or a project that needs
+                          coding. I’d love to help with it</h3>
```

Comment:
[Optional]

- [ ] It would be semantically correct if you use `p` here instead of `h3` since it's a paragraph 💯